modules/virt-updating-virtio-drivers-windows.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+:_content-type: PROCEDURE
+[id="virt-updating-virtio-drivers-windows_{context}"]
+= Updating VirtIO drivers on a Windows guest
+
+Update the `virtio` drivers by using Windows Update.
+
+.Prerequisites
+
+* A Windows guest OS with the `virtio` drivers must be attached to the VM.
+
+.Procedure
+
+. In the Windows Guest OS, use the `File Explorer` to navigate to *Settings* -> *Windows Update* -> *Advanced Options* -> *Optional Updates*
+. Install all updates from *Red Hat, Inc.*.
+. Reboot the virtual machine (VM).
+
+.Verification
+
+. On the Windows VM, navigate to the *Device Manager*
+. Select a device.
+. Select the *Driver* tab.
+. Click *Driver Details*.
+.. Confirm that the `virtio` driver details list the correct version.
```

Comment:
I think you could combine the last 2 steps - "Click Driver Details to confirm that the virtio driver has been updated."